bindings/python/iree/runtime/vm.cc

```diff
@@ -690,6 +690,30 @@ void SetupVmBindings(pybind11::module m) {
       .def_property_readonly(
           "stashed_flatbuffer_blob",
           [](VmModule& self) { return self.get_stashed_flatbuffer_blob(); })
+      .def_property_readonly(
+          "function_names",
+          [](VmModule& self) {
+            py::list names;
+            iree_vm_module_signature_t sig =
+                iree_vm_module_signature(self.raw_ptr());
+            for (size_t ordinal = 0; ordinal < sig.export_function_count;
+                 ++ordinal) {
+              iree_vm_function_t f;
+              iree_string_view_t linkage_name;
```

Comment:
I forgot this linkage_name thing existed - I can remove that now as it should always match what the iree_vm_function_name returns

Reply:
Yeah. Mind removing it after I land this :)  It is already elsewhere in this file, so already need to scalpel it.